src/pages/[platform]/build-a-backend/functions/examples/custom-message/index.mdx

```diff
@@ -0,0 +1,86 @@
+import { getCustomStaticPath } from '@/utils/getCustomStaticPath';
+
+export const meta = {
+  title: 'Custom message',
+  description:
+    'Use an Auth custom message authentication trigger to customize the message sent to users',
+  platforms: [
+    'android',
+    'angular',
+    'flutter',
+    'javascript',
+    'nextjs',
+    'react',
+    'react-native',
+    'swift',
+    'vue'
+  ]
+};
+
+export function getStaticPaths() {
+  return getCustomStaticPath(meta.platforms);
+}
+
+export function getStaticProps() {
+  return {
+    props: {
+      meta
+    }
+  };
+}
+
+You can use `defineAuth` and `defineFunction` to create a [Cognito custom message Lambda trigger](https://docs.aws.amazon.com/cognito/latest/developerguide/user-pool-lambda-custom-message.html) that sends an custom email or phone verification message or a multi-factor authentication (MFA) code.
+
+To get started, install `@types/aws-lambda` package that will be used to define the type of the handler:
+
+```bash title="Terminal" showLineNumbers={false}
+npm install --save @types/aws-lambda
+```
+
+
+Next, create a new directory and a resource file, `amplify/auth/custom-message/resource.ts`. Then, define the function with defineFunction:
+
+```ts title="amplify/auth/custom-message/resource.ts"
+import { defineFunction } from '@aws-amplify/backend';
+
+export const myCustomMessageFunction = defineFunction({
+  entry: './handler.ts',
+});
+```
+
+Next, create the corresponding handler file, `amplify/auth/custom-message/handler.ts`, file with the following contents:
+
+```ts title="amplify/auth/custom-message/handler.ts" /* spell-checker: disable */
```

Comment:
is the spell checker disable comment needed?

Reply:
I think you'll need to "renable" it after disabling so that cspell can check the rest of the page: https://cspell.org/configuration/document-settings/#enable--disable-checking-sections-of-code

So you would need to wrap your codeblock with

```
/* spell-checker: disable */
... code here ...
/* spell-checker: enable */
```